library/psa_crypto.c

```diff
@@ -1303,11 +1423,14 @@ static psa_status_t psa_set_key_policy_internal(
 static psa_status_t psa_start_key_creation(
     const psa_key_attributes_t *attributes,
     psa_key_handle_t *handle,
-    psa_key_slot_t **p_slot )
+    psa_key_slot_t **p_slot,
+    psa_se_drv_table_entry_t **p_drv )
 {
     psa_status_t status;
     psa_key_slot_t *slot;
 
+    *p_drv = NULL;
```

Comment:
We do need this because `psa_validate_persistent_key_parameters` is called conditionally.

It is currently the case that `if( p_drv != NULL ) *p_drv = NULL;` in `psa_validate_persistent_key_parameters` will never trigger, but those ~4 bytes of code are worth it to make the function more robust. Uninitialized pointers can be nasty.